pkg/validate/validate.go

```diff
@@ -31,6 +31,8 @@ var (
 		"BaseSpecPath":      validateInstallPackagePath,
 		"CustomPackagePath": validateInstallPackagePath,
 		"DefaultNamespace":  validateDefaultNamespace,
+		"TrafficManagement.Components.Proxy.Common.Values.includeIPRanges": validateIPRanges,
+		"TrafficManagement.Components.Proxy.Common.Values.excludeIpRanges": validateIPRanges,
```

Comment:
All values are validated in validate_values.go - please move this PR there.
These fields are already validate there but I think "*" is not checked.

Reply:
Thanks for clarification! Updated.